mail_quoted_reply/models/mail_compose_message.py

```diff
@@ -6,18 +6,28 @@
 class MailComposeMessage(models.TransientModel):
     _inherit = "mail.compose.message"
 
-    @api.onchange("template_id")
-    def _onchange_template_id_wrapper(self):
-        super()._onchange_template_id_wrapper()
-        context = self._context
-        if "is_quoted_reply" in context.keys() and context["is_quoted_reply"]:
-            self.body += Markup(context["quote_body"])
-        return
+    @api.depends("composition_mode", "model", "res_domain", "res_ids", "template_id")
+    def _compute_body(self):
+        res = super()._compute_body()
+        for composer in self:
+            context = composer._context
+            if context.get("is_quoted_reply"):
```

Comment:
since it appears to depend on a context key, aren't we supposed to add this key in `@api.depends_context()` ?